extensions/tmtv/src/utils/measurementServiceMappings/utils/getSOPInstanceAttributes.js

```diff
@@ -4,6 +4,8 @@ export default function getSOPInstanceAttributes(imageId) {
   if (imageId) {
     return _getUIDFromImageID(imageId);
   }
+
+  return {};
```

Comment:
I used `measurementService.getMeasurements` and it gave all the measurements (including those that are empty and with no SOP). We should keep them separate as they are non-exportable and we don't have support for SR for SCOORD3D

Reply:
Yes good catch, thank you so much. I will make updates.